src/components/atoms/OcList/OcList.vue

```diff
@@ -9,11 +14,35 @@ export default {
   name: "OcList",
   status: "ready",
   released: "unreleased",
+  props: {
+    raw: {
```

Comment:
Could I ask you to add examples for this component, both for the raw and non-raw version?

Reply:
yes sure

Reply:
added gerneral docs for oclist, raw="true" doesnt seem to have an effect inside ODS docs (in web it does)